test/s3_test.rb

```diff
@@ -184,8 +184,8 @@ def filesystem
       @s3.client.stub_responses(:put_object, -> (context) {
         @s3.client.stub_responses(:get_object, content_disposition: context.params[:content_disposition])
       })
-      @s3.upload(fakeio, "foo", content_disposition: 'inline; filename=""été bar.pdf""')
-      assert_equal "inline; filename=\"\"été bar.pdf\"\"", CGI.unescape(@s3.object("foo").get.content_disposition)
+      @s3.upload(fakeio, "foo", content_disposition: 'inline; filename=""été foo bar.pdf""')
+      assert_equal "inline; filename=\"\"été foo bar.pdf\"\"", CGI.unescape(@s3.object("foo").get.content_disposition)
```

Comment:
This doesn't actually test the escaping behavior, but I just thought the filenames should be consistent.